Add `SetReturnType` and `Asyncify` types (#127)
sindresorhus/type-fest: readme.md

- [`UnionToIntersection`](source/union-to-intersection.d.ts) - Convert a union type to an intersection type.
- [`Stringified`](source/stringified.d.ts) - Create a type with the keys of the given type changed to `string` type.
- [`FixedLengthArray`](source/fixed-length-array.d.ts) - Create a type that represents an array of the given type and length.

> - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters of a given function type.

Author: ```suggestion
- [`SetReturnType`](source/set-return-type.d.ts) - Create a function type by modifying the return type of an existing function type.
```

What do you think?
